dll/win32/shell32/shlfileop.cpp

```diff
@@ -1219,6 +1219,34 @@ static void destroy_file_list(FILE_LIST *flList)
     HeapFree(GetProcessHeap(), 0, flList->feFiles);
 }
 
+static CString* try_find_new_name(LPCWSTR szDestPath)
+{
+    CString *mask = new CString(szDestPath);
```

Comment:
Please use wide-string equivalent here and in other places (`CStringW` for example).